.vale.ini

```diff
@@ -17,3 +17,7 @@ Microsoft.Passive = NO
 Microsoft.SentenceLength = NO
 Microsoft.Vocab = NO
 Microsoft.We = NO
+
+TokenIgnores = ({{ .*? -?}})
+proselint.Typography = NO
+Microsoft.Ellipses = NO
```

Comment:
Aware that this isn't something we want, but I'm not sure how to disable this for just `.chglog/CHANGELOG.tpl.md`. In that file, I think it makes sense for an ellipsis to be made up of individual periods so that it's possible to use it in Git as a range immediately, e.g. `2.28.0..HEAD`.